data/resources/style.css

```diff
@@ -160,29 +160,30 @@ messageindicators {
 }
 
 messagelabel {
-  margin: 6px 12px;
-  font-size: 0.95em;
+  margin: 6px 9px;
+  line-height: 1.1;
```

Comment:
This sometimes removes the underline from links that are broken up into several lines
![Bildschirmfoto vom 2022-06-04 21-13-02](https://user-images.githubusercontent.com/3630213/172022439-67f1e375-6398-45fe-bb94-d676c2cf862f.png)

Reply:
Yeah, this indeed seems to break the underlines a lot more than usual (even though they are not always perfect even without it). I'll remove the line-height commit for now, maybe it's better to just wait for https://gitlab.gnome.org/GNOME/libadwaita/-/issues/365.